client/src/nl/sensorlab/videowall/animation/baseanimations/liquidcolumns/LiquidColumn.java

```diff
@@ -0,0 +1,61 @@
+package nl.sensorlab.videowall.animation.baseanimations.liquidcolumns;
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PApplet;
+import processing.core.PGraphics;
+import processing.core.PVector;
+
+public class LiquidColumn {
+
+	private LiquidColumnsAnimation parent; 
+
+	private PVector[] positions;
+	private float speed;
+
+	public LiquidColumn(LiquidColumnsAnimation p, float x, float speed) {
+		this.parent = p;
+		this.positions = new PVector[parent.PIXEL_RESOLUTION_Y];
+		this.speed = speed;
+		for(int y = 0; y < parent.PIXEL_RESOLUTION_Y; y++) {
+			this.positions[y] = new PVector(x, y);
+		}
+	}
+
+	public void update() {
+		// Ignore first node
```

Comment:
Why? It's useful to future reference to document these so it makes debugging later easier.